benchbuild/utils/run.py

```diff
@@ -5,6 +5,8 @@
 from contextlib import contextmanager
 
 import attr
+import plumbum as pb
+from mypy_extensions import DefaultArg, VarArg
```

Comment:
Codacy found an issue: [Unable to import 'mypy_extensions'](https://app.codacy.com/manual/polyjit/benchbuild_2/pullRequest?prid=5694346)